src/main/java/oss/fosslight/api/controller/v1/ApiProjectController.java

```diff
@@ -277,13 +277,17 @@ public CommonResult createProject(
 			if(CoConstDef.CD_OPEN_API_CREATE_PROJECT_LIMIT > createCnt) {
 				Map<String, Object> paramMap = new HashMap<String, Object>();
 				
-				String osTypeStr = CoCodeManager.getCodeString(CoConstDef.CD_OS_TYPE, osType);
-				
-				if(isEmpty(osTypeStr)) {
-					return responseService.getFailResult(CoConstDef.CD_OPEN_API_PARAMETER_ERROR_MESSAGE
-							, CoCodeManager.getCodeString(CoConstDef.CD_OPEN_API_MESSAGE, CoConstDef.CD_OPEN_API_PARAMETER_ERROR_MESSAGE));
+				if (!isEmpty(osType)) {
+					String osTypeStr = CoCodeManager.getCodeString(CoConstDef.CD_OS_TYPE, osType);
+
+					if (isEmpty(osTypeStr)) {
+						return responseService.getFailResult(CoConstDef.CD_OPEN_API_PARAMETER_ERROR_MESSAGE
+								, CoCodeManager.getCodeString(CoConstDef.CD_OPEN_API_MESSAGE, CoConstDef.CD_OPEN_API_PARAMETER_ERROR_MESSAGE));
+					}
+				} else {
+					osType = CoConstDef.CD_OS_TYPE_LINUX;
```

Comment:
You can get the list of operation system type by using "getValues" function of oss.fosslight.common.CoCodeManager.
Then, osType can be set as the first item of the list instead of hard-coding.

Also, you should remove "require=true" property in OS Type API param and releated codes.